plugin.php

```diff
@@ -573,12 +577,25 @@ function rest_handle_options_request( $response, $handler, $request ) {
 
 		foreach ( $endpoints as $endpoint ) {
 			$accept = array_merge( $accept, $endpoint['methods'] );
+
+			$methods = array_keys( $endpoint['methods'] );
+			$method = array_shift( $methods );
```

Comment:
Why are endpoints registered in this format?

```
'methods' =>
    array (
      'DELETE' => true,
    ),
```

When would a method be `false`, or an endpoint registered supporting two methods?